cmake/module/GenerateSetupNsi.cmake

```diff
@@ -5,8 +5,8 @@
 function(generate_setup_nsi)
   set(abs_top_srcdir ${PROJECT_SOURCE_DIR})
   set(abs_top_builddir ${PROJECT_BINARY_DIR})
-  set(PACKAGE_URL ${PROJECT_HOMEPAGE_URL})
-  set(PACKAGE_TARNAME "bitcoin")
+  set(CLIENT_URL ${PROJECT_HOMEPAGE_URL})
+  set(CLIENT_TARNAME "bitcoin")
```

Comment:
In b75d532ca4703e26f59c7d33a882a6c1295784ed: The `CLIENT_TARNAME` re-naming is incomplete.

Reply:
Thanks! Fixed.